libs/datasets/timeseries.py

```diff
@@ -183,7 +183,7 @@ def demographic_distributions_by_field(
                 # variables with different buckets for a given distribution, including none's
                 # will mix buckets.  To properly fix, consider passing latest in a long format
                 # rather than a wide variables format.
-                if value is None:
+                if value is None or np.isnan(value):
```

Comment:
`np.nan` values are not (any longer?) equal to `None`, so empty items were not being caught by this statement. I've left the `None` logic in place in case there are actual `None` values, but we could probably just use  `if np.isnan(value):` by itself.